certify-sunbird-insurance.properties

```diff
@@ -1,41 +1,43 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+## ------------------------------------------- Plugin enable properties ------------------------------------------------------------
+mosip.certify.integration.scan-base-package=io.mosip.certify.sunbirdrc.integration
+mosip.certify.integration.vci-plugin=SunbirdRCVCIssuancePlugin
+mosip.certify.integration.audit-plugin=LoggerAuditService
 
+## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
+mosip.certify.vciplugin.sunbird-rc.enable-psut-based-registry-search=false
+mosip.certify.vciplugin.sunbird-rc.issue-credential-url=http://10.3.148.107/credential/credentials/issue
+mosip.certify.vciplugin.sunbird-rc.supported-credential-types=LifeInsuranceCredential,InsuranceCredential,VehicleInsuranceCredential
 
-## -------------------------------------- Authentication & Authorization -----------------------------------------------
+  ##---------------------------------Life Insurance Credential Configurations------------------------------------------------------
```

Comment:
Lets have this as comment without --- line so the sections are not confused

Reply:
added as a comment